objectVariable/view.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE HTML>
+<html>
+	<head>
+		<title>vipin mavi</title>
+		<script type="text/javascript" src="css/css.css"></script>
```

Comment:
Please include **.css** file not **.js**

Reply:
done